package.json

```diff
@@ -51,7 +51,7 @@
   },
   "homepage": "https://github.com/daostack/arc.js#readme",
   "dependencies": {
-    "@daostack/migration": "0.0.0-alpha.56-v3",
+    "@daostack/migration": "0.0.0-alpha.58-v2",
```

Comment:
backwards?

Reply:
@tibetsprague no, the earlier one is 56, the latter, 58